data/core/macros/special-notes.cfg

```diff
@@ -1,125 +1,323 @@
 #textdomain wesnoth-help
 #define SPECIAL_NOTES
-    _"
+#deprecated 1 Use [special_note]note= tags instead to apply special notes, or use the {SPECIAL_NOTE_*} macros (note the singular).
+    "
 
-Special Notes:"#enddef
+"+_"Special Notes:"#enddef
+
+# SPECIAL_NOTES_* are the deprecated versions that provide the strings.
+# SPECIAL_NOTE_* are the new versions that provide an entire tag.
+# For now, the new are defined in terms of the old, but when adding new special notes,
+# only a SPECIAL_NOTE_* form should be added.
 
 #define SPECIAL_NOTES_SPIRIT
-_"
-• Spirits have very unusual resistances to damage, and move quite slowly over open water."#enddef
+#deprecated 1 Use {SPECIAL_NOTE_SPIRIT} instead (note the singular), which generates a [special_note] tag.
```

Comment:
:thinking: 

I don't really like the idea of having `{SPECIAL_NOTES_FOO}` and `{SPECIAL_NOTE_FOO}`, the names are too similar. I feel like we're condemning some UMC author in the future to hours of hair-pulling to figure out why they are getting a weird error when they add a special note to their unit. These macros _are_ few, as you say, but they're pretty commonly used: 43% of the units in ageless have special notes.

I don't have an alternative name to suggest, sorry.

Reply:
Yeah, I understand what you mean. Unfortunately, I don't think it's possible to craft a macro that would work both in the `""+{...}` context and also on a line by itself.

Reply:
I'm not proposing to create a macro that would work in both contexts. I'm proposing to name the new macros differently so the names of the old and new macro won't be as easy to conflate with each other.